src/SFA.DAS.RoATPService.Database/SFA.DAS.RoATPService.Database.sqlproj

```diff
@@ -77,6 +77,7 @@
     <Build Include="Tables\OrganisationCategory.sql" />
     <Build Include="Tables\OrganisationCategoryOrgTypeProviderType.sql" />
     <Build Include="Tables\OrganisationStatusEvent.sql" />
+    <Build Include="StoredProcedures\RoATP_FAT_Data_Export.sql" />
```

Comment:
you can remove this like from the proj, it won't change behavior but keep the proj cleaner

Reply:
doesn't it need to be there for the Dacpac deploy?

Reply:
yes it 100% needs to be there

Reply:
ignore me then

Reply:
the other stored procedures are in this list.